docs/integration/voice.md

```diff
@@ -4,27 +4,63 @@ This section describes what is required to add a PLAY command which will be unde
 
 The core HA voice intents support NEXT TRACK, PREVIOUS TRACK, PAUSE, UNPAUSE and VOLUME to a specific player or to an area. HA does not intend to add any more media player service calls at this time so you will need to use custom sentences to cover any other of your requirements. See this [discussion for how](https://github.com/orgs/music-assistant/discussions/2176).
 
-## LLM Conversation Agent
+!!! note
+    Queue behaviour when adding items by Assist will follow the settings in MA SETTINGS>>CORE>>PLAYER QUEUES CONTROLLER.
+    
+## HA Assist
+![easiest label](../assets/label-easiest.png)
+
+The Music Assistant Integration allows the use of custom intents for initiating playback that are kept completely local. As with all HA intents the structure of the request is strict.  
 
 **Installation Requirements**
 
-- You need to be running HA 2024.6.0 or later
 - You need to be running the latest version of the HA Integration
-- You need a voice assistant which uses a LLM AI conversation agent configured (even if you want to just type in the query)
-- Preferably allow the auto-exposure of MA media players to Assist (otherwise you will have to do that manually)
+- You need a voice assistant configured (even if you want to just type in the query) (Examples: [Cloud Pipeline](https://www.home-assistant.io/voice_control/voice_remote_cloud_assistant/) or [Local Pipeline](https://www.home-assistant.io/voice_control/voice_remote_local_assistant/))
+- Add a directory to your Home Assistant `config` directory named `custom_sentences/en`
```

Comment:
Can we also simply create copies of this file for other languages ? I would love to also use this in my own language, Dutch.

Reply:
Yes. You found that Discord thread where it is discussed. Once we get more than English I can change the text here,

Reply:
Yes, need to get stuff translated and then it would go in the corresponding custom_sentences/(2char-lang-name) folder